### ISSUE TITLE
Specify Monitor

### ISSUE THREAD

**Issue by tgaldi** [LittleVaaty/VDesk#17]
Is there a way to specify the monitor on a desktop/virtual desktop to launch the application?

**Comment by LittleVaaty** [comment]
No, I have look into it, and i have not found an easy of doing it. 
I plan to do it, but i will be later

**Comment by tgaldi** [comment]
Would it be easy to specify screen coordinates in the short term? You would have to calculate it manually to position on other monitors, but that should work.

**Comment by LittleVaaty** [comment]
If i remember correctly, screen coordinates should be possible

**Comment by Ami-Mizuno** [comment]
you can always enumerate the monitors installed 

1. Use EnumDisplayMonitors to get a list of displays (do not pass an HDC to use entire VScreen)
2. In the enum callback build your list. you can use a custom structure that contains a number value and the LPRECT arg in the enum callback.
3. when vdesk needs to move an open app the user can pass a number for the display and vdesk will use the stored rect. 

You can make it start at the upper left or wherever. just remember that the rect is in virtual screen coords so x,y can be any value (including negative) and are not always going to be 0,0

that should get you started. 

in case you dont already have it here's a clicky to some MSDN help on the topic
https://learn.microsoft.com/en-us/windows/win32/gdi/multiple-display-monitors-functions

**Comment by LittleVaaty** [comment]
Thank, i didn't know about that function.
I will look into it but that should work

**Comment by Ami-Mizuno** [comment]
No worries remmy. Great work by the way 👍

**Comment by Rimbaldo** [comment]
@LittleVaaty Are there any news on development on specifying the monitor in which any app should open??

Thanks!

**Comment by LittleVaaty** [comment]
I'm currently working on the process problem and after that i will work on this.
I haven't had much time recently, but I should be able to get back to it soon.
